core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -386,7 +384,8 @@ SslContextFactory getSslContextFactory() {
     return this.sslContextFactory;
   }
 
-  private void configureConnectors(SslContextFactory sslContextFactory) {
+  private void configureConnectors(SslContextFactory sslContextFactory,
+                                   List<NamedURI> namedListeners) {
```

Comment:
This go back to the comment I made in the constructor. Here, we parse the non-named listeners inside the method, but named-listeners have to passed as an argument. I find it inconsistent.